lib/ui/src/containers/menu.tsx

```diff
@@ -221,6 +235,7 @@ export const useMenu = (
       prev,
       next,
       collapse,
+      ...getAddonsShortcuts(),
```

Comment:
@apapadopoulou looks like this is where the shortcuts get added?

Reply:
Yes, thank you!